pkg/controller/kieapp/kieapp_controller_test.go

```diff
@@ -787,7 +787,7 @@ func TestConsoleLinkCreation(t *testing.T) {
 
 	consoleLink := &consolev1.ConsoleLink{}
 	reconciler.Service.Get(context.TODO(), getNamespacedName("", getConsoleLinkName(cr)), consoleLink)
-	assert.Equal(t, "Business Central", consoleLink.Spec.Text)
+	assert.Equal(t, "cr: Business Central", consoleLink.Spec.Text)
```

Comment:
might this be better? to test new func?
```golang
assert.Equal(t, getConsoleLinkFriendlyName(cr), consoleLink.Spec.Text)
```

Reply:
I don't think so because the function can be wrong and return always "foo" and the test would still pass